### PR TITLE
Fix task failed to apply

### LINKED ISSUE #217: 7.1.0 - > 7.2.0
Bumped version from 7.1.0 to 7.2.0 and getting:

No value has been specified for property 'android$ktlint_gradle'.

Am using kotlin jvm.  

Build is here: https://github.com/nwillc/ksvg/blob/master/build.gradle.kts

### PR DESCRIPTION
**Pull request by Tapchicoma**
Fixes #217 

Includes setup for `7.2.1` release.

### CODE REVIEW

File: plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt [hunk 1]
```diff
@@ -382,6 +382,7 @@ open class KtlintPlugin : Plugin<Project> {
             description = "Format all .kts files in project directory " +
                 "to ensure that they are formatted according to ktlint."
             configurePluginTask(target, extension, ktLintConfig) {
+                android.set(false)
```
Comment (JLLeitschuh) [inline plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt]: Why would someone want to not use the android format for their scripts?
Shouldn't we just honor the flag universally?

Comment (Tapchicoma) [reply plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt]: Good question :thinking: My opinion was somehow biased that android style guide should be only applied for android project sources, but not for kotlin script files.

I would remove that.

Comment (JLLeitschuh) [reply plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt]: I think I agree that removing it and going with the universal configuration is better. Alternatively, offer a separate configuration flag.

Comment (Tapchicoma) [reply plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt]: Removed this.

**Comment by Tapchicoma** [comment]
Just need to update Gradle in plugin and samples to the latest version, test is not needed.

**Pull request merged**